Assets/Scripts/Gameplay/SetBomb.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class SetBomb : MonoBehaviour
+{
+    [SerializeField]
+    private GameObject _bombPrefab;
+    [SerializeField]
+    private Canvas m_Clock;
+
+    private void Start()
+    {
+        AetherInput.GetPlayerActions().SetBomb.performed += HandleSetBomb;
+    }
+
+    private void HandleSetBomb(InputAction.CallbackContext context)
+    {
+        Instantiate(_bombPrefab, transform.position, transform.rotation);
+        m_Clock.GetComponent<CanvasGroup>().alpha = 1;
```

Comment:
Don't call `GetComponent` outside of `Start` and `Awake` unless absolutely necessary. This is bad for performance as all components in the gameobject has to be iterated. Consider caching the `CanvasGroup` object in in `Start` or `Awake`